tooling/nargo/src/ops/test.rs

```diff
@@ -113,8 +113,11 @@ pub fn run_test<B: BlackBoxFunctionSolver<FieldElement>>(
                 {
                     use acvm::acir::circuit::Program;
                     use noir_fuzzer::FuzzedExecutor;
+                    use proptest::test_runner::Config;
                     use proptest::test_runner::TestRunner;
-                    let runner = TestRunner::default();
+
+                    let runner =
+                        TestRunner::new(Config { failure_persistence: None, ..Config::default() });
```

Comment:
Did you try using either of the following `failure_persistence` options?
- `FileFailurePersistence::WithSource`
- `FileFailurePersistence::Direct`
?

It appears that the `SourceParallel` option looks for `.rs` source files, which might explain why it wasn't finding the `proptest-regressions` folder.

Reply:
I didn't try it because I assume that whoever wrote this didn't intend to persist these.

Reply:
Yeah, I think for the purposes of this pull request just silencing these errors is correct. We may be deprecating this fuzzer due to keshas work anyway.